packages/@react-aria/focus/stories/FocusScope.stories.tsx

```diff
@@ -107,31 +108,26 @@ export function Example({isPortaled, contain}: StoryProps) {
 
 function FocusableFirstInScopeExample() {
   let [contentIndex, setContentIndex] = useState(0);
-  let [buttonRemoved, setButtonRemoved] = useState(false);
   function DialogContent(index = 0) {
-    const nextIndex = index === 2 ? 0 : index + 1;
+    const nextIndex = index + 1;
     return (
-      <>
+      <div key={index}>
         <h1 id={`heading-${index}`}>Dialog {index + 1}</h1>
-        {index === 2 ?
+        {index > 2 ?
```

Comment:
This case is not showing in storybook example. There are [3 instances of Dialog content](https://github.com/adobe/react-spectrum/pull/3393/files#diff-9dbac8303c8771335c1fa9cfb18cb9ef8b95fba4a0492fbc78ead85cf216aea6L138), so the "end of the road" statement should come when `index === 2`.
```suggestion
        {index === 2 ?
```